cmd/pbstatuser/main.go

```diff
@@ -134,7 +134,7 @@ func checkSourceAvailabilityAWS(ctx context.Context) {
 			sr := kafka.SourceResult{
 				SourceID:     s.SourceID,
```

Comment:
I believe we also need to change the ID to application ID :thinking:

Reply:
I believe it is returned from sources as ResourceID here: https://github.com/RHEnVision/provisioning-backend/blob/0c4591ecb299ea9e42c3036508bf74145966c427/internal/clients/http/sources/sources_client.go#L136 
Tho we would need to pass it down to Authentication to get it here I guess :thinking:

Reply:
I have the JSON [label](https://github.com/RHEnVision/provisioning-backend/blob/0c4591ecb299ea9e42c3036508bf74145966c427/internal/kafka/source_result.go#L17 ) set to `resoure_id`, is it enough or do you prefer changing the field to ResourceID instead of SourceID?

Reply:
We can add the ResourceID to the authentication [struct](https://github.com/RHEnVision/provisioning-backend/blob/0c4591ecb299ea9e42c3036508bf74145966c427/internal/clients/authentication.go#L11) and then use it here: https://github.com/RHEnVision/provisioning-backend/blob/0c4591ecb299ea9e42c3036508bf74145966c427/cmd/pbstatuser/main.go#L89. 
Does it make sense?

Reply:
Yeah, that is the best way I've though of as well :+1: